src/types/Quote.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @module TypeInterfaces/quote
+ */
+/**
+ * Dummy comment needed for correct doc display, do not remove.
+ */
+
+import ICType from './CType'
+
+export interface ICostBreakdown {
+  tax: number
+  net: number
+  gross: number
```

Comment:
`gross` or `net` could be calculated from the other two. Also @Dudleyneedham had the idea to enable the user to further break down the tax (i.e there could be multiple different taxes).

Reply:
I have updated as per our tallk